src/ess-experimentation/handler.js

```diff
@@ -0,0 +1,477 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* c8 ignore start */
+import RUMAPIClient from '@adobe/spacecat-shared-rum-api-client';
+import { JSDOM } from 'jsdom';
+import { AuditBuilder } from '../common/audit-builder.js';
+
+const EXPERIMENT_PLUGIN_OPTIONS = {
+  experimentsRoot: '/experiments',
+  experimentsConfigFile: 'manifest.json',
+  experimentsMetaTag: 'experiment',
+  experimentsQueryParameter: 'experiment',
+};
+const SPACECAT_RUM_API_ENDPOINT = 'https://spacecat.experiencecloud.live/api/v1/rum';
```

Comment:
we decided not to expose the rum stuff yet due to some constraints. This endpoint doesn't exist currently. For now, we can just set the `fullAuditRef` to domain.